Dockerfile

```diff
@@ -7,9 +7,12 @@ RUN apt-get -y update && \
 	cd manticore && \
 	pip install .
 
-RUN useradd -m manticore
-USER manticore
-WORKDIR /home/manticore
-ENV HOME /home/manticore
+RUN apt-get install -y build-essential software-properties-common && \
+    add-apt-repository -y ppa:ethereum/ethereum && \
+    apt-get update && \
+    apt-get install -y solc ethereum
```

Comment:
what does installing the `ethereum` package get us?

Reply:
i noticed that it does add some time to the docker build stage, but then i realized docker hub will generally be doing that, not our users directly, so i don't think it's a big deal, nvm.